src/pages/Create/components/Creator.tsx

```diff
@@ -183,11 +185,7 @@ export default function Creator() {
           />
         </div>
 
-        {vestingDuration !== 0 && (
-          <GovnftChart startDate={selectedStartDate} vestingDuration={vestingDuration} cliffDuration={cliffDuration} />
-        )}
-
-        {isAddress(toAddress) && amount && Number(vestingDuration) !== 0 && (
+        {isAddress(toAddress) && amount && Number(vestingDuration) !== 0 && description && (
```

Comment:
Stas mentioned we should not require them to put a description, thats why I removed it